testsuite/integration-tests/src/test/resources/org/jboss/resteasy/test/core/basic/BasicWeb.xml

```diff
@@ -0,0 +1,12 @@
+<web-app version="3.0" xmlns="http://java.sun.com/xml/ns/javaee"
```

Comment:
Can you please rename this file to NoApplicationSubclassWeb.xml? To be clear that this web.xml is related with NoApplicationSubclassTest class ...

Reply:
Done.